internal/analytics/segment_reporter.go

```diff
@@ -333,3 +359,11 @@ func (r *SegmentReporter) getNodeCount(ctx context.Context, k8sCli kubernetes.In
 
 	return workerNodesCount, controlPlaneNodesCount, nil
 }
+
+func generatePluginReport(plugin config.Plugin, pluginName string, pluginType plugin.Type) pluginReport {
+	return pluginReport{
+		Name: pluginName,
+		Type: pluginType,
+		RBAC: plugin.Context.RBAC,
```

Comment:
As I mentioned earlier, we should anonymize the reporting. That means we should remove anything what could identify the user.
Like:
- rbac group name
- rbac user name

BTW I'd suggest to convert this to a method, to scope it to the Segment reporter.